cardano-cli/src/Cardano/CLI/Byron/Delegation.hs

```diff
@@ -16,8 +16,6 @@ where
 import           Cardano.Api.Byron
 import qualified Cardano.Api.Ledger as L
```

Comment:
What are we exporting from `Cardano.Api.Ledger`?

Reply:
Apparently these three only:
- [`Annotated(..)`](https://cardano-ledger.cardano.intersectmbo.org/cardano-ledger-binary/Cardano-Ledger-Binary-Decoding.html#t:Annotated)
- [`serialize'`](https://cardano-ledger.cardano.intersectmbo.org/cardano-ledger-binary/Cardano-Ledger-Binary-Encoding.html#v:serialize-39-)
- [`byronProtVer`](https://cardano-ledger.cardano.intersectmbo.org/cardano-ledger-binary/Cardano-Ledger-Binary-Version.html#v:byronProtVer)